RELEASING.md

```diff
@@ -1,14 +1,13 @@
 Releasing
 =========
 
- 1. Change the version in `gradle.properties` and `mkdocs.yml` to a non-SNAPSHOT verson.
```

Comment:
im assuming you have to update the version in gradle props still when doing a release?

Reply:
Oh woops yes 🤦 I missed that part